dispatchr-ruby/app/models/request_item.rb

```diff
@@ -1,4 +1,9 @@
 class RequestItem < ApplicationRecord
   belongs_to :request, optional: true
   belongs_to :item, optional: true
+
+  validates :request_id, presence: true
+  validates :item_id, presence: false
```

Comment:
it was true originally but realized it didn't need to be there. doesnt hurt to have it?